test/e2e/config/azure-dev.yaml

```diff
@@ -51,7 +51,9 @@ providers:
       - sourcePath: "${PWD}/templates/test/cluster-template-prow-ipv6.yaml"
         targetName: "cluster-template-ipv6.yaml"
       - sourcePath: "../data/infrastructure-azure/cluster-template-mhc.yaml"
-        targetName: "cluster-template-mhc.yaml"        
+        targetName: "cluster-template-mhc.yaml"
+      - sourcePath: "${PWD}/templates/test/cluster-template-machine-pool.yaml"
```

Comment:
the file name is `cluster-template-machinepool.yaml` and not `cluster-template-machine-pool.yaml`

the e2e is failing because cannot find the file